aspnetcore/signalr/hubs.md

```diff
@@ -158,6 +158,7 @@ If an exceptional condition must be propagated to the client, use the <xref:Micr
 * <xref:signalr/introduction>
 * <xref:signalr/javascript-client>
 * <xref:signalr/publish-to-azure-web-app>
+* [SIGNALR ON THE WIRE – AN INFORMAL DESCRIPTION OF THE SIGNALR PROTOCOL](https://blog.3d-logic.com/2015/03/29/signalr-on-the-wire-an-informal-description-of-the-signalr-protocol/)
```

Comment:
This blog post is for the older System.Web version of SignalR. For the ASP.NET Core SignalR protocol, I'd link to https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/docs/specs/HubProtocol.md which @BrennanConroy did. We should probably make an aka.ms link in case we move folders or something.